.github/workflows/dependency-check.yml

```diff
@@ -20,10 +20,15 @@ jobs:
       - name: Check vulnerabilities
         uses: sonatype-nexus-community/nancy-github-action@b492e6567a301a914bd227d44bf10dcfe85438ad
         with:
-          nancyCommand: sleuth -e bba60acb-c7b5-4621-af69-f4085a8301d0,d373dc3f-aa88-483b-b501-20fe5382cc80,5def94e5-b89c-4a94-b9c6-ae0e120784c2,dcf6da03-f9dd-4a4e-b792-0262de36a0b1
+          nancyCommand: sleuth -e bba60acb-c7b5-4621-af69-f4085a8301d0,d373dc3f-aa88-483b-b501-20fe5382cc80,5def94e5-b89c-4a94-b9c6-ae0e120784c2,dcf6da03-f9dd-4a4e-b792-0262de36a0b1,c16fb56d-9de6-4065-9fca-d2b4cfb13020
           # Check the dependency by entering the OSS index UUID here:
           # https://ossindex.sonatype.org/vuln/bba60acb-c7b5-4621-af69-f4085a8301d0
           #
           # To get more detail locally run `go list -json -m all | nancy sleuth`
           # dcf6da03-f9dd-4a4e-b792-0262de36a0b1 is because of gogo/protobuf@1.3.1
           # which is used by go-libp2p-core, need them to upgrade to 1.3.2 before we can remove it.
+          #
+          # c16fb56d-9de6-4065-9fca-d2b4cfb13020
+          # See https://github.com/dgrijalva/jwt-go/issues/463
+          # If that happens to get released in a 3.X.X version, we can add a constraint to our go.mod
+          # for it. If its in 4.X.X, then we need all our transitive deps to upgrade to it.
```

Comment:
ah actually https://github.com/smartcontractkit/chainlink/pull/4108/commits/fe1de6d6ff610d0eba0f8045c76b5fff7178087e also fixes this
```
replace github.com/dgrijalva/jwt-go => github.com/form3tech-oss/jwt-go v3.2.1+incompatible
```
not sure if that's typical to do in go

Reply:
oh yeah I like that better (actually fixes the vuln)

Reply:
👍🏼